README.md

```diff
@@ -51,7 +68,13 @@ Add to the datasource configuration:
 
 ### With `kumactl`
 
-If you use `kumactl install metrics` with a version of kumactl >= 1.3.0 the plugin will be setup automatically.
+#### kumactl >= 1.7.0
+
+If you use `kumactl install observability --components grafana,prometheus` with a version of kumactl >= 1.7.0 the plugin will be setup automatically.
```

Comment:
Don't specify the components. It's just better if jaeger is there as well.

Reply:
so this applies probably to everything, right?

Reply:
yes